### ISSUE TITLE
lsBox created with wrong dimensions in Python Module

### ISSUE THREAD

**Issue by XaverKlemenschits** [ViennaTools/ViennaLS#27]
The following code does not produce the expected box:
```
import viennaLS3d as vls

extent = 30
gridDelta = 0.5

bounds = (-extent, extent, -extent, extent, -extent, extent)
boundaryCons = (0, 0, 1) # 0 = reflective, 1 = infinite, 2 = periodic
mask = vls.lsDomain(bounds, boundaryCons, gridDelta)
minCorner = (-10, -10,0)
maxCorner = (10, 10, 4)
vls.lsMakeGeometry(mask, vls.lsBox(minCorner, maxCorner)).apply()
mesh = vls.lsMesh()
vls.lsToMesh(mask, mesh).apply()
vls.lsVTKWriter(mesh, f"minimal.vtk").apply()
```

If the lsBox is not created inline, but an object is created, it works as expected:
```
box = vls.lsBox(minCorner, maxCorner)
vls.lsMakeGeometry(mask, box).apply()
```

**Comment by XaverKlemenschits** [comment]
Most likely to do with return value policies of pybind11:
https://pybind11.readthedocs.io/en/stable/advanced/functions.html#keep-alive]

**Comment by XaverKlemenschits** [comment]
Fixed in #29 by changing to copy passed geometries rather than references.
